package.json

```diff
@@ -7,31 +7,23 @@
     "lint:fix": "eslint --fix ."
   },
   "devDependencies": {
-    "@ethereum-waffle/mock-contract": "^3.1.0",
     "@nomiclabs/buidler": "^1.4.7",
     "@nomiclabs/buidler-ethers": "^2.0.0",
     "@nomiclabs/buidler-waffle": "^2.1.0",
     "@openzeppelin/contracts": "^3.2.0",
-    "@types/chai": "^4.2.3",
-    "@types/ethereumjs-abi": "^0.6.3",
-    "@types/lodash": "^4.14.161",
+    "@types/chai": "^4.2.12",
     "@types/mocha": "^8.0.3",
     "@typescript-eslint/eslint-plugin": "^4.1.1",
     "@typescript-eslint/parser": "^4.1.1",
     "@uniswap/v2-core": "^1.0.1",
-    "bn.js": "^5.1.3",
     "chai": "^4.2.0",
     "eslint": "^7.9.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-no-only-tests": "^2.4.0",
     "eslint-plugin-prettier": "^3.1.4",
     "ethereum-waffle": "^3.1.0",
-    "ethereumjs-abi": "^0.6.8",
-    "ethereumjs-util": "^7.0.5",
-    "ethers": "5.0.14",
-    "lodash": "^4.17.20",
```

Comment:
No more lodash!? I don't know what it was ever for, but I am glad its gone now!

Reply:
It was used for `_.deepClone` before.